src/openqaoa-core/openqaoa/algorithms/qaoa/qaoa_workflow.py

```diff
@@ -450,6 +450,21 @@ def evaluate_circuit(
         # if the backend is the analytical simulator, we just return the expectation value of the cost Hamiltonian
         if isinstance(self.backend, QAOABackendAnalyticalSimulator):
             output_dict.update({"cost": self.backend.expectation(params_obj)[0]})
+        # if the workflow implements SPAM Twirling, 
+        # we just return the expectation value of the cost Hamiltonian and measurement outcomes
+        elif isinstance(self.backend, SPAMTwirlingWrapper):
+            cost = self.backend.expectation(params_obj)
+            measurement_results = (
+                self.backend.measurement_outcomes
+                if isinstance(self.backend.measurement_outcomes, dict)
+                else self.backend.measurement_outcomes.tolist()
+            )
+            output_dict.update(
+                {
+                    "cost": cost,
```

Comment:
let's change the key `cost` to `expectation` in the `output_dict` for consistency with the docstring of the function

Reply:
Wouldn't it make more sense to change the docstring t `cost` instead?
I checked the implications of this change and the docstring is the only bit that mentions "uncertainty" as a field, or do you think the name `expectation` would be much clearer

Reply:
Either is fine! You can make the choice, the main point is just consistency